txscript/data/script_tests.json

```diff
@@ -2792,5 +2792,27 @@
 ["0 0x01 0x50", "HASH160 0x14 0x77c33ee1f1892b124f2c9aaed62114c23cffe2da EQUAL", "NONE", "ERR_RESERVED_OPCODE", "OP_RESERVED in P2SH should fail"],
 ["0 0x01 VER", "HASH160 0x14 0xe682277f7669fe986d315d52b56df3d244da11a0 EQUAL", "NONE", "ERR_RESERVED_OPCODE", "OP_VER in P2SH should fail"],
 
+["STAKE opcodes test "],
+
+["SSTX related tests"],
+["1 1 ADD", "SSTX 2 OP_EQUAL", "NONE", "OK", "SSTX is a NOP"],
+["DATA_5 DATA_4 0x7a{4}", "SSTX HASH160 DATA_20 0x246bda0f2b9aba6dc78c6259970e45e6af960d46 EQUAL", "NONE", "ERR_EVAL_FALSE", "SSTX almost P2SH"],
+["DATA_5 1 3 DATA_2 ADD SSTX", "HASH160 DATA_20 0x30c7ecf9ff5508c105071194d979b13ed90a82b7 EQUAL", "NONE", "OK", "SSTX (P2SH)"],
```

Comment:
As with #1381, these aren't all actually testing the intended thing, rather they're just pushing data to the stack.